draft-ietf-anima-brski-prm.md

```diff
@@ -1298,27 +1342,29 @@ The header of the RVR SHALL contain the following parameter as defined for JWS {
 
```

Comment:
> (It optionally contains the certificate chain for this certificate)

For the RVR, I would expect that the certificate chain MUST be included, otherwise the MASA needs to keep track of all possible Domain Registrars and obtain and manage their certificate chains for them... I propose

> it MUST also contain the certificate chain for this certificate for verification at the MASA

Reply:
Agree, this would be a meaningful restriction. We intended to be open, but I agree, it would make the MASA life easier, if it always gets the full path.

Reply:
Done